Nc-News/src/components/Article.jsx

```diff
@@ -23,20 +28,39 @@ export function Article() {
     );
   } else {
     return (
-      <article>
-        <h1>{article.title}</h1>
-        <p className="author">{article.author + " in " + article.topic}</p>
-        <p className="posted">
-          {created.toLocaleString("en-GB").split(",")[0]}
-        </p>
-        <img
-          src={article.article_img_url}
-          alt={article.title + " Killer Image"}
-        />
-        <p className="body">{article.body}</p>
-        <p className="upvotes">{article.votes + " Upvotes"}</p>
-        <p className="comments">{article.comment_count + " Comments"}</p>
-      </article>
+      <>
+        <article>
+          <h1>{article.title}</h1>
+          <p className="author">{article.author + " in " + article.topic}</p>
+          <p className="posted">
+            {created.toLocaleString("en-GB").split(",")[0]}
+          </p>
+          <img
+            src={article.article_img_url}
+            alt={article.title + " Killer Image"}
+          />
+          <p className="body">{article.body}</p>
+          <p className="upvotes">{article.votes + " Upvotes"}</p>
+          <Link
+            className="comments"
+            to={"/articles/" + article_id + "/comments"}
+          >
+            {article.comment_count} Comments
+          </Link>
+        </article>
+        <hr className="commentPreviews" />
+        <section className="commentPreviews">
+          <CommentCard comment={comments[0]} />
+          <CommentCard comment={comments[1]} />
+          <CommentCard comment={comments[2]} />
```

Comment:
This functionality breaks when there are no comments - or probably less than 3 comments
You can check this using article 37 which has no comments
Also these comments have no eggs 🥚 , whereas the comments in your comments route do 🤔 Can't quite see why - perhaps css is affecting this page differently